test/definitions/scenarios/upgrade_satellite_6_2_test.rb

```diff
@@ -10,6 +10,7 @@ module Scenarios
       end
       assume_feature_present(:foreman_tasks)
       stub_foreman_proxy_config
+      stub_package_manager
```

Comment:
As per offline discussion: probably makes sense to move this stubbing to test helper